packages/compass/src/app/styles/deprecated-bootstrap-styles/dropdowns.less

```diff
@@ -58,6 +58,11 @@
   box-shadow: 0 6px 12px fade(@palette__black, 18%);
   background-clip: padding-box;
 }
+
+.dropdown-menu::-webkit-scrollbar-track {
+  background-color: @palette__white;
```

Comment:
Is this the change from the https://github.com/mongodb-js/compass/pull/3581/files pr? Not sure how it made it in here

Reply:
Yeah, not sure what git magic I've employed to mess this up